package.json

```diff
@@ -1,114 +1,21 @@
-    "build-module": "npm run clean-module && tsc -p tsconfig.json",
-    "build-browser": "npm run clean-browser && webpack --config webpack.browser.js --env.env=prod",
-    "analyze-bundle": "npm run clean-browser && webpack --config webpack.config.js --env.env=analyze",
-    "lint": "eslint \"src/**/*.ts\" && eslint \"__tests__/**/*.ts\"",
-    "lint-fix": "eslint --config .eslintrc.json --fix \"src/**\" && eslint --config .eslintrc.json --fix \"__tests__/**/*.ts\"",
-    "format": "prettier --write \"src/**\" \"__tests__/**\"",
-    "licenses": "license-checker --production --failOn \"AGPL-1.0-only; AGPL-1.0-or-later; AGPL-3.0-only; AGPL-3.0-or-later; Beerware; CC-BY-NC-1.0; CC-BY-NC-2.0; CC-BY-NC-2.5; CC-BY-NC-3.0; CC-BY-NC-4.0; CC-BY-NC-ND-1.0; CC-BY-NC-ND-2.0; CC-BY-NC-ND-2.5; CC-BY-NC-ND-3.0; CC-BY-NC-ND-4.0; CC-BY-NC-SA-1.0; CC-BY-NC-SA-2.0; CC-BY-NC-SA-2.5; CC-BY-NC-SA-3.0; CC-BY-NC-SA-4.0; CPAL-1.0; EUPL-1.0; EUPL-1.1; EUPL-1.1;  GPL-1.0-only; GPL-1.0-or-later; GPL-2.0-only;  GPL-2.0-or-later; GPL-3.0; GPL-3.0-only; GPL-3.0-or-later; SISSL;  SISSL-1.2; WTFPL\"",
-    "dev": "tsc-watch --preserveWatchOutput --noClear",
-    "dev-server": "concurrently \"npm run dev\" \"npm run dev-test-helper\" \"cd examples/server && npm run dev\"",
-    "dev-script": "concurrently \"npm run dev\" \"npm run dev-test-helper\" \"cd examples/script && npm run dev\"",
-    "dev-bundle": "concurrently \"npm run dev\" \"npm run dev-test-helper\" \"cd examples/bundle && npm run dev\"",
-    "dev-ios": "concurrently \"npm run dev\" \"npm run dev-test-helper\" \"cd examples/native && npm run dev-ios\"",
-    "dev-android": "concurrently \"npm run dev\" \"npm run dev-test-helper\" \"cd examples/native && npm run dev-android\"",
-    "dev-test-helper": "cd examples/test-helper && npm run dev",
     "test": "npm run lint && npm run licenses && npm run test-unit",
-    "test-unit": "jest --coverage --verbose",
-    "test-acceptance": "npm run test-acceptance-server && npm run test-acceptance-script && npm run test-acceptance-bundle && npm run test-acceptance-ios && npm run test-acceptance-android",
-    "test-acceptance-server": "echo \"Not Implemented\" && exit 1",
-    "test-acceptance-script": "echo \"Not Implemented\" && exit 1",
-    "test-acceptance-bundle": "echo \"Not Implemented\" && exit 1",
-    "test-acceptance-ios": "echo \"Not Implemented\" && exit 1",
-    "test-acceptance-android": "echo \"Not Implemented\" && exit 1",
-    "bootstrap-examples": "./bootstrap-examples.bash",
-    "build-api-docs": "npx typedoc --out docs/api/source/api",
-    "build-docs-preview-site": "npm run build-api-docs; cd docs/prose; make html; cd ../api; make html; cd ../; rm -r dist || true; mkdir -p dist/prose dist/api; cp -r prose/build/html/. dist/prose/; cp -r api/build/html/. dist/api/; echo 'Draft documentation: <a href=\"./prose/\">Prose</a>, <a href=\"./api/\">API docs</a>.' >> dist/index.html"
+    "test-unit": "jest --coverage --verbose"
```

Comment:
This came from a suggestion I read somewhere when Googling around - i.e. that `lerna run test`, which is fine too of course, can be slow, as it kicks off a full, independent Jest test process in each sub-package. This option can (apparently) be quicker, as it runs a single Jest process to run all tests across all packages within the one process. It might also be a way to generate a single coverage report for the entire repo, as opposed to multiple coverage reports, one per-package.
But on both these points I'd need to play around with it more to confirm and see if it does actually add value, but for that we'll need to actually have multiple packages first!

Reply:
I'm not too worried about tests running slow (when iterating, you can always just run the tests from just the project you're working on), and individual coverage reports don't seem problematic to me either (as long as the build just fails if one of them does not meet the threshold). But I'd make this dependent indeed in how much trouble this actually gives you when you actually have multiple packages - which is why I'd do the split right away as well :)